### DISCUSSION TITLE
ERROR: Out of memory for dense node cache, reduce --cache size.

### DISCUSSION THREAD

**Discussion by Vrumfondel** [osm-search/Nominatim#2289] [Q&A]
Hi there,

I tried to upgrade my 3.4 installation to 3.7 - basically decided to re-install from scratch.
Installation as such seemed to be fine so far, but I fail in the initial map import with

"ERROR: Out of memory for dense node cache, reduce --cache size."

I found some solutions for the older config-structure, but how to change the cache-size in the .env file?

**Comment by lonvia** [comment]
Use the `--osm2pgsql-cache` parameter for ` nominatim import`. Please read the [Notes on memory usage](https://nominatim.org/release-docs/latest/admin/Import/#notes-on-memory-usage) for best use of this parameter.
